app/controllers/concerns/bookmark_common.rb

```diff
@@ -0,0 +1,9 @@
+module BookmarkCommon
```

Comment:
Usually these are in the Foreman::Controller namespace, see other files in the same dir.